apps/search/js/private_browsing.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+/* global newtab */
+
+(function(exports) {
+
+  function PrivateBrowsing() {
+    this.trigger = document.getElementById('private-window');
+    this.trigger.addEventListener('click', this);
+  }
+
+  PrivateBrowsing.prototype = {
+
+    /**
+     * General event handler.
+     * Currently dispatches a request to open a new private window
+     * from the newtab object.
+     */
+    handleEvent: function(e) {
+      newtab.requestPrivateWindow();
```

Comment:
Even though we have only 1 event at the moment, I still think we should verify the event type before opening the private window. It's more readable and make things easier to add events in the future.